css/style.css

```diff
@@ -1,19 +1,166 @@
+body {
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+}
+
+/* HEADER */
+header {
+  padding: 20px;
+}
 
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+nav {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
 
-body {
-    font-family: 'Roboto', sans-serif;
-    -webkit-font-smoothing: antialiased;
+.logo {
+  width: 30px;
+  height: 30px;
+}
+
+ul.nav-list {
+  display: none;
+  list-style: none;
+  padding: 0 20px;
+  color: black;
+}
+
+a.nav-item {
+  text-decoration: none;
+  cursor: pointer;
+  padding: 0 10px;
+  color: gray;
+}
+
+.nav-item:hover {
+  color: rgb(250, 115, 66);
+}
+
+/* HERO SECTION */
+.hero {
+  height: 350px;
+  background-image: url("../img/first-background.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  color: white;
+  text-align: center;
+  font-weight: lighter;
+}
+
+.hero h1 {
+  font-size: 2.3rem;
+  font-weight: lighter;
+  padding: 70px 20px 0 20px;
+  margin: 0;
+}
+
+.hero p {
+  font-size: 1.2rem;
+  padding: 0 65px 20px 65px;
+  line-height: 2rem;
+}
+
+.hero button {
+  width: 120px;
+  padding: 0.7rem;
+  background-color: rgb(250, 115, 66);
+  border: none;
+  border-radius: 4px;
+}
+
+/* ABOUT */
+.about {
+  color: rgb(82, 80, 80);
+  text-align: center;
+  padding: 40px 60px 10px 60px;
+}
+
+.about h2 {
+  font-weight: lighter;
```

Comment:
Usually best to be more precise here. You've got font weights 400, 500 and 300 loaded into the project (see line 11 of index.html) so you can use those values, example: `font-weight: 300`

Reply:
Good point.  Will change that also, thank you